.github/workflows/membership-frontend-build.yml

```
@@ -0,0 +1,112 @@
+name: Build membership-frontend
+
+on:
+    pull_request:
+        branches:
+            - main
+    push:
+        branches:
+            - main
+
+jobs:
+    membership_frontend_build:
+        if: >-
+            (github.actor != 'dependabot[bot]') &&
+              (github.repository_owner == 'guardian' ||
+                github.event_name == 'push')
+
+        # Required by actions-assume-aws-role
+        permissions:
+            id-token: write
+            contents: read
+
+        name: membership-frontend build
+        runs-on: ubuntu-latest
+        steps:
+            - name: Env
+              run: env
+
+            - name: Set Time Zone to GMT # Setting the Timezone as the test cases except it to be to Europe/London(for the daylight saving changes #
+              run: |
+                sudo timedatectl set-timezone Europe/London
+
+            - name: Checkout repo
+              uses: actions/checkout@v3
+
+        # (Respects .nvmrc)
+            - name: Setup Node
+              uses: guardian/actions-setup-node@v2.4.1
+              with:
+                  cache: "yarn"
+                  cache-dependency-path: frontend/yarn.lock
+
+            - name: Run Grunt and Install yarn
+              run: |
+                set -o xtrace
+                set -o nounset
+                set -o errexit
+                export YARN_VERSION="1.12.3"
+                export YARN_LOCATION="tools/${YARN_VERSION}"
+                if [ ! -d "$YARN_LOCATION" ]; then
+                mkdir -p ${YARN_LOCATION}
+                cd ${YARN_LOCATION}/
+                wget -qO- https://github.com/yarnpkg/yarn/releases/download/v${YARN_VERSION}/yarn-v${YARN_VERSION}.tar.gz | tar zvx
+                cd ../..
+                fi
+              working-directory: frontend
+
+            - name: Install Dependencies
+              run: |
+                  export YARN_VERSION="1.12.3"
+                  export YARN_LOCATION="tools/${YARN_VERSION}"
+                  ${YARN_LOCATION}/yarn-v${YARN_VERSION}/bin/yarn cache clean
+                  ${YARN_LOCATION}/yarn-v${YARN_VERSION}/bin/yarn install
+              working-directory: frontend
```

Yarn v1.22.19 [is installed by default on the runners](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md). Could we check whether this works with that version of yarn? If it does, we can also remove some of this code.
```suggestion
            - name: Install Dependencies
              run: |
                  yarn cache clean
                  yarn install
              working-directory: frontend
```

---

This worked with the  default yarn version on the runners.So have changed. as per the suggestion